src/pages/Details/Details.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import BookIntroduction from './BookIntroduction/BookIntroduction';
+import UserReviews from './UserReviews/UserReviews';
+import ReviewsList from './ReviewsList/ReviewsList';
+import BookContainer from './BookContainer/BookContainer';
+
+import './Details.scss';
+
+class Details extends Component {
+  constructor() {
+    super();
+    this.state = {
+      bookDetail: [],
+      url: './data/DetailDataV2.json',
+    };
+  }
+
+  componentDidMount() {
+    const { url } = this.state;
+    fetch(url)
+      .then(res => res.json())
+      .then(output => {
+        this.setState({
+          bookDetail: output.message[0],
+        });
+      });
+  }
+
+  render() {
+    const { bookDetail } = this.state;
+
+    return (
+      <article className="totalBox">
+        {bookDetail.product_info && (
+          <BookContainer bookDetail={bookDetail.product_info} />
+        )}
+        {bookDetail.product_info && (
+          <BookIntroduction
+            bookDetail={bookDetail.product_info}
+            title="작품소개"
+            contents={bookDetail.product_info.description}
+          />
+        )}
+        {bookDetail.product_info && (
+          <BookIntroduction
+            bookDetail={bookDetail.product_info}
+            title="목차"
+            contents={bookDetail.product_info.index}
+          />
+        )}
+        {bookDetail.product_info && (
+          <UserReviews bookDetail={bookDetail.product_info} />
+        )}
```

Comment:
```suggestion
        {bookDetail.product_info && (
          <BookContainer bookDetail={bookDetail.product_info} />
          <BookIntroduction
            bookDetail={bookDetail.product_info}
            title="작품소개"
            contents={bookDetail.product_info.description}
          />
          <BookIntroduction
            bookDetail={bookDetail.product_info}
            title="목차"
            contents={bookDetail.product_info.index}
          />
          <UserReviews bookDetail={bookDetail.product_info} />
       )}
```
bookDetail.product_info 라는 동일한 조건이 부여된다면 따로 나누지 않고 한 번에 작성해주세요!

Reply:
해당 부분 아직 적용이 안되어 있네요!